packages/core/src/browser/tree/tree.ts

```diff
@@ -233,6 +238,7 @@ export class TreeImpl implements Tree {
 
     protected _root: TreeNode | undefined;
     protected readonly onChangedEmitter = new Emitter<void>();
+    protected readonly onNodesAddedEmitter = new Emitter<TreeNode[]>();
```

Comment:
I think this file should be reverted, there are already `FileNavigatorModel.refresh` to ensure that nodes are added and `FileNavigatorWidget.revealFile` which ensures and reveal.

Reply:
done